rocAL_pybind/amd/rocal/types.py

```diff
@@ -108,6 +108,10 @@
 from rocal_pybind.types import TRIMTOSHAPE
 from rocal_pybind.types import ERROR
 
+#     RocalMelScaleFormula
+from rocal_pybind.types import SLANEY
```

Comment:
please rename to MEL_FILTER_XX

Reply:
Renamed to MELSCALE_SLANEY and MELSCALE_HTK